README.md

```diff
@@ -63,20 +63,20 @@ https://central.sonatype.com/artifact/com.sap.cds/cds-feature-attachments
 
 ## Artifactory
 
-SAP internally, snapshots are deplyoed to SAP Artifactory:
-https://int.repositories.cloud.sap/ui/repos/tree/General/proxy-cap-java
+Snapshots are deployed to SAP's Artifactory in DMZ:
+https://common.repositories.cloud.sap/artifactory/cap-java/com/sap/cds/cds-feature-attachments/
 
-If you have access to this repository and want to use the plugin you need to add this repository to the
-maven `settings.xml`.
+The snapshots are also replicated to SAP's internal Artifactory:
+https://int.repositories.cloud.sap/artifactory/proxy-cap-java/com/sap/cds/cds-feature-attachments/
```

Comment:
Avoid a ref to an internal repo in this public doc.

Reply:
removed